pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -63,80 +55,128 @@ def __init__(
         self.control_columns = control_columns
         self.adstock_max_lag = adstock_max_lag
         self.yearly_seasonality = yearly_seasonality
+        self.date_column = date_column
+        self.validate_data = validate_data
+
         super().__init__(
-            data=data,
-            target_column=target_column,
             date_column=date_column,
             channel_columns=channel_columns,
-            channel_prior=channel_prior,
-            validate_data=validate_data,
+            model_config=model_config,
+            sampler_config=sampler_config,
             adstock_max_lag=adstock_max_lag,
         )
 
-    def _preprocess_channel_prior(self) -> TensorVariable:
-        return (
-            pm.HalfNormal.dist(sigma=2, shape=len(self.channel_columns))
-            if self.channel_prior is None
-            else change_dist_size(
-                dist=self.channel_prior, new_size=len(self.channel_columns)
-            )
-        )
+    @property
+    def default_sampler_config(self) -> Dict:
+        return {"progressbar": True, "random_seed": 1234}
```

Comment:
I don't think we should provide a fixed `random_seed` by default
```suggestion
        return {"progressbar": True}
```

The progressbar also seems unnecessary, since that's the default anyway?

Reply:
yes, each of the child classes need to implement it's own default sampler_config. In model builder it's an abstract function. If you don't think here any settings should be passed I can make it into an empty dict

Reply:
Sounds like it doesn't need to be an `abstract` property in the base class, in that case? 
Usually we can use PyMC sampler defaults just fine

Reply:
I'd say that I'm open to all cosmetics being changed in the future, but as a patch to model builder. Now I'd focus on all things that would be deal-breakers to finally merge it. Then we can focus on tuning and small updates

Reply:
now default_sampler_config = {}